tests/by-util/test_touch.rs

```diff
@@ -917,3 +917,27 @@ fn test_touch_reference_symlink_with_no_deref() {
     // Times should be taken from the symlink, not the destination
     assert_eq!((time, time), get_symlink_times(&at, arg));
 }
+
+#[test]
+fn test_obsolete_posix_format() {
+    let (at, mut ucmd) = at_and_ucmd!();
+    ucmd.env("_POSIX2_VERSION", "199209")
+        .env("POSIXLY_CORRECT", "1")
+        .args(&["01010000", "11111111"])
+        .succeeds()
+        .no_output();
+    assert!(at.file_exists("11111111"));
+    assert!(!at.file_exists("01010000"));
+}
+
+#[test]
+fn test_obsolete_posix_format_with_year() {
```

Comment:
Hm, I'm unable to reproduce the behavior of this test with GNU `touch 9.6`:
```
$ _POSIX2_VERSION=199209 POSIXLY_CORRECT=1 touch 9001010000 111
$ ls
111  9001010000
```

Reply:
Ok, I think I figured it out: the year must be at the end according to https://www.gnu.org/software/coreutils/manual/html_node/touch-invocation.html